deps/rabbitmq_amqp1_0/src/rabbit_amqp1_0_reader.erl

```diff
@@ -695,6 +687,18 @@ auth_phase_1_0(Response,
               handshake, 8)
     end.
 
+track_channel(Channel, ChFrPid, State) ->
+  rabbit_log:debug("AMQP 1.0 opened channel = ~tp " , [{Channel, ChFrPid}]),
+  State#v1{tracked_channels = maps:put(Channel, ChFrPid, State#v1.tracked_channels)}.
+untrack_channel(Channel, State) ->
+  case maps:take(Channel, State#v1.tracked_channels) of
+    {Value, NewMap} -> rabbit_log:debug("AMQP 1.0 closed channel = ~tp ", [{Channel, Value}]),
+                       State#v1{tracked_channels = NewMap};
+    error -> State
+  end.
+find_session_pid_for_channel(Channel, State) ->
```

Comment:
newline before new function. Not sure this function is necessary either as getting an item from a map is trivial